app/src/main/java/dev/arkbuilders/rate/data/db/PortfolioDao.kt

```diff
@@ -0,0 +1,72 @@
+package dev.arkbuilders.rate.data.db
+
+import androidx.room.Dao
+import androidx.room.Entity
+import androidx.room.Insert
+import androidx.room.OnConflictStrategy
+import androidx.room.PrimaryKey
+import androidx.room.Query
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.map
+import dev.arkbuilders.rate.domain.model.Asset
+import dev.arkbuilders.rate.domain.model.CurrencyCode
+import dev.arkbuilders.rate.domain.repo.PortfolioRepo
+import javax.inject.Inject
+import javax.inject.Singleton
+
+@Entity
+data class RoomAsset(
```

Comment:
Please consider extracting these into smaller files and put in particular package. For example:
```bash
data
|--db
     |--entity
     |--dao
|--network
     |--dto
     |--api
repo
```